packages/million/types.ts

```diff
@@ -34,113 +34,129 @@ export abstract class AbstractBlock {
   abstract parent(): HTMLElement | null | undefined;
 }
 
-export const enum Edits {
-  TYPE,
-  NAME,
-  VALUE,
-  HOLE,
-  INDEX,
-  LISTENER,
-  PATCH,
-  BLOCK,
+export const enum Flags {
+  CHILD = 1,
+  EVENT = 2,
+  ATTRIBUTE = 4,
+  STYLE_ATTRIBUTE = 8,
+  SVG_ATTRIBUTE = 16,
+  BLOCK = 32,
 }
```

Comment:
Does esbuild support cross-file inline const enum now? If not, using `export const enum` will increase the bundle size and decrease the performance.